ci/test-checks.sh

```diff
@@ -78,6 +78,7 @@ nightly_clippy_allows=()
    --deny=warnings \
    --deny=clippy::default_trait_access \
    --deny=clippy::integer_arithmetic \
+   --deny=clippy::used_underscore_binding \
```

Comment:
Just adding a comment here with ref to clippy-lint description: https://rust-lang.github.io/rust-clippy/master/#used_underscore_binding